resources/views/Widgets/Common/CollapseWidget.twig

```diff
@@ -0,0 +1,28 @@
+{% set collapseBoxes = widget.settings.collapseBox.mobile %}
+{% set collapseId    = uid() %}
+{% set renderContent = false %}
+
+{% for collapseBox in collapseBoxes if collapseBox.headline is not empty %}
+    {% set renderContent = true %}
+{% endfor %}
+
+{% if renderContent %}
+    <div class="widget accordion widget-{{ widget.settings.appearance.mobile | default('primary')  }}" id="accordion_{{ collapseId }}">
+        {% for collapseBox in collapseBoxes %}
+        <div class="card">
+          <button v-toggle-faq-icon="{newIcon: 'fa-chevron-up', oldIcon: 'fa-chevron-down'}" class="card-header btn btn-link" id="heading_{{ collapseId }}_{{ loop.index }}" type="button" data-toggle="collapse" data-target="#collapse_{{ collapseId }}_{{ loop.index }}" aria-expanded="false" aria-controls="collapse_{{ collapseId }}_{{ loop.index }}">
```

Comment:
Bootstrap setzt die Klasse `.collapsed` auf das `button`-Element. Die Klasse kann verwendet werden, um das Icon zu tauschen oder per `transform: rotate(180deg)` zu drehen. In Kombination mit `transition` bekommt man sogar eine Art Animation hin.

Reply:
done